typhos/positioner.py

```diff
@@ -222,7 +222,7 @@ def _get_timeout(self, set_position, settle_time):
         else:
             acc_time = acc_sig.get()
         # This time is always greater than the kinematic calc
-        return abs(delta/speed) + 2 * abs(acc_time) + abs(settle_time)
+        return rescale * (abs(delta/speed) + 2 * abs(acc_time)) + abs(settle_time)
```

Comment:
If we're not going to include settle time maybe it's worth explaining ourself a bit in the docstring here... though I guess it's not really a public method.  I feel like it's a 50/50 shot someone will think the `rescale` applies to `settle_time` as well

Reply:
I'll write docstrings, they were glaringly missing prior to this edit and I was hoping I wouldn't need to but I should just do it

My thought was that you wouldn't want your two input args to interfere with each other